test/ast.spec.js

```diff
@@ -466,6 +466,11 @@ describe('AST', () => {
             expect(getParsedSql(sql)).to.equal('SELECT NULL');
         });
 
+        it('should support trailing zeros', function() {
+            expect(getParsedSql('SELECT 042')).oneOf(['SELECT 42', 'SELECT 042']);
```

Comment:
I'm curious: what is the use case for this?

Why do you test the result w/ and w/o leading zeros? Shouldn't it be always the same result?

Reply:
I think the parser should be allowed to either drop the leading zeros or leave them as they are. Form my understanding both are correct behavior.

Reply:
The parser must produce the same result for the same input. In this case it should remove the leading zero. Please replace `oneOf` with `equal` and test for number w/o leading zeros.

Reply:
Done.